openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -138,58 +138,53 @@
 - control_key: AU-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+      text: |      
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
+ 
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-162
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
```

Comment:
OCP uses fluentd and does not inherit settings from the underlying operating system's audit daemon.

When cluster logging is enabled, to meet AU-5(1), we'll need to create a response that outlines how fluentd (or another means) can be configured to alert system owners when disk partitions containing logs begin to get full.

Control ref:
https://nvd.nist.gov/800-53/Rev4/control/AU-5#enhancement-1

AU-5(1) | RESPONSE TO AUDIT PROCESSING FAILURES \| AUDIT STORAGE CAPACITYThe information system provides a warning to [Assignment: organization-defined personnel, roles, and/or locations] within [Assignment: organization-defined time period] when allocated audit record storage volume reaches [Assignment: organization-defined percentage] of repository maximum audit record storage capacity.
-- | --